docs/guide/troubleshooting.md

```diff
@@ -31,3 +31,7 @@ module.exports = {
 
 ::: warning
 Disabling `resolve.symlinks` may break hot module reloading if your dependencies are installed by third-party npm clients that utilized symbolic links, such as`cnpm` or `pnpm`.
+
+::: 'vue' is not recognized as an internal or external command
+This error happens in Windows if you don't have npm in your system path. To check: from a command prompt type `path`.
+Confirm the actual path to your npm repo. It should be like `c:\Users\[YourLoginName]\AppData\Roaming\npm` (using your actual login name). Add that to your System Properties, Environment Variables.
```

Comment:
You get this path by using command: `> npm config get prefix`

Had this problem yesterday too.